src/api/localStorage.js

```diff
@@ -0,0 +1,96 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('UserAccounts')) || [];
+  }
+
+  createUser(user) {
+    this.updateUsers();
+    user.id = this.generateId();
+    this.users.push(user);
+    this.setLocalStorageData();
+  }
+
+  setLocalStorageData() {
+    localStorage.setItem('UserAccounts', JSON.stringify(this.users));
+  }
+
+  updateUsers() {
+    this.users = JSON.parse(localStorage.getItem('UserAccounts')) || [];
+  }
+
+  getUser(id) {
+    const user = this.users.find(user => String(user.id) === String(id));
+    return user;
+  }
+
+  getAllUsers() {
+    return this.users;
+  }
+
+  editUser(id, editName, editValue) {
+    try{
+      const editObj = {};
+      editObj[`${editName}`] = `${editValue}`;
+      const updatedUser = Object.assign(this.getUser(id), editObj);
+      this.deleteUser(id);
+      this.createUser(updatedUser);
+    } catch (err) {console.log('Error Editing User: ', err);}
+  }
+
+  replaceUser(id, update) {
+    const user = this.users.find(user => String(user.id) === String(id));
+    const replacedUser = Object.assign(user, update);
```

Comment:
if you're not doing anything with it beyond returning, skip the `const` assignment and just return it
```suggestion
    return Object.assign(user, update);
```